core/trino-main/src/main/java/io/trino/execution/SqlTaskManager.java

```diff
@@ -201,33 +188,6 @@ private QueryContext createQueryContext(
                 localSpillManager.getSpillSpaceTracker());
     }
 
-    @Override
-    public synchronized void updateMemoryPoolAssignments(MemoryPoolAssignmentsRequest assignments)
-    {
-        if (coordinatorId != null && coordinatorId.equals(assignments.getCoordinatorId()) && assignments.getVersion() <= currentMemoryPoolAssignmentVersion) {
-            return;
-        }
-        currentMemoryPoolAssignmentVersion = assignments.getVersion();
-        if (coordinatorId != null && !coordinatorId.equals(assignments.getCoordinatorId())) {
-            log.warn("Switching coordinator affinity from %s to %s", coordinatorId, assignments.getCoordinatorId());
-        }
-        coordinatorId = assignments.getCoordinatorId();
-
-        for (MemoryPoolAssignment assignment : assignments.getAssignments()) {
-            if (assignment.getPoolId().equals(GENERAL_POOL)) {
-                queryContexts.getUnchecked(assignment.getQueryId()).setMemoryPool(localMemoryManager.getGeneralPool());
```

Comment:
is this a good reason to combine this with https://github.com/trinodb/trino/pull/11071 together as single change?

Reply:
Yeah - could be. I wanted to keep them separate for sake of easier review (the followup is mostly a code cleanup with no user-visible changes). But I may squash them together if you prefer.

Reply:
i reviewed https://github.com/trinodb/trino/pull/11071 instead of this one

Reply:
I will close this one then